docker-compose.yml

```diff
@@ -25,10 +35,10 @@ services:
   ipfs0:
     container_name: ipfs0
     image: ipfs/go-ipfs:release
-    ports:
-          - "4001:4001" # ipfs swarm
-#         - "5001:5001" # expose if needed/wanted
-#         - "8080:8080" # exposes if needed/wanted
+#    ports:
+#         - "4001:4001" # ipfs swarm - expose if needed/wanted
```

Comment:
what changed?

Reply:
I think it does not make a difference for the testing purposes, since it's not a node that will be dialed from the outside